lib/main.dart

```diff
@@ -36,8 +36,7 @@ class MyApp extends StatelessWidget {
     NetworkService netConnection = Network();
     SoundService sound = Sounds();
     AnalyticsService analytics = Analytics(_firebaseAnalytics);
-    GameApisService gameApi = Games(analytics: analytics);
-    AdsService adsData = AdsService(analytics: analytics, sound: sound);
+    AdsService adsData = AdsService();
```

Comment:
don't we need `Analytics` and `Sound` inside the `AdsService`? what is wrong with the previous method?

Reply:
We had to use integration class with onUpdate ads callback

Reply:
What is ad data?
Why continue this issue after multiple mentions

Reply:
This has gotten fixed two days ago on the devel branch [Add `IntegratedServices`] commit.

Reply:
Please select  better name
Thanks